source/BcmI2cController.h

```diff
@@ -265,8 +265,8 @@ class BcmI2cControllerClass : public I2cControllerClass
     const ULONG _DIV_USED_MASK = 0x0000FFFF;  // Mask of non-reserved bits in the Clock Divider Register
 
     // Standard (100khz) and high (400khz) divider values.
-    const ULONG CDIV_100KHZ = 1500;
-    const ULONG CDIV_400KHZ = 376;      // Actually 398.9khz, divider value must be even
+    const ULONG CDIV_100KHZ = 2500;
```

Comment:
If the original value worked (for something, I assume), would there be a selector to get the appropriate values for that device?  If not, how did we test it before (or did it never work?).